README.md

```diff
@@ -11,17 +11,16 @@
 | [![](https://img.shields.io/badge/docs-dev-blue.svg)](https://juliamanifolds.github.io/Manifolds.jl/latest/) | [![CI](https://github.com/JuliaManifolds/Manifolds.jl/workflows/CI/badge.svg)](https://github.com/JuliaManifolds/Manifolds.jl/actions?query=workflow%3ACI+branch%3Amaster) | [![DOI](https://zenodo.org/badge/190447542.svg)](https://zenodo.org/badge/latestdoi/190447542) |
 | | [![codecov.io](http://codecov.io/github/JuliaManifolds/Manifolds.jl/coverage.svg?branch=master)](https://codecov.io/gh/JuliaManifolds/Manifolds.jl/) |
 
-
 Package __Manifolds.jl__ aims to provide both a unified interface to define and
 use manifolds as well as a library of manifolds to use for your projects.
-This package is under development, and subject to changes as needed.
+This package is mostly stable, see https://github.com/JuliaManifolds/Manifolds.jl/issues/438 for planned upcoming changes.
 
 ## Getting started
 
 To install the package just type
 
 ```julia
-] add Manifolds
+using Pkg; Pkg.add("Manifolds")
```

Comment:
I feel the `] add Manifolds` is nicer, but you are right, it always is marked as an error in markdown :/

Reply:
I think the main advantage of `using Pkg; Pkg.add("Manifolds")` is that it can be directly pasted into the command line but Markdown not marking it as an error is also a nice benefit :slightly_smiling_face: .

Reply:
Ok.